journal/test-journal.sh

```diff
@@ -8,48 +8,48 @@ set -o pipefail
 
 # Skip empty lines
 ID=$(journalctl --new-id128 | sed -n 2p)
->/expected
+>/opt/expected
```

Comment:
what is the issues was in the atomic mode so that we moved to /opt. I guess /tmp would be a better place since we are already writing the tests results/outputs to /tmp dir. Also describe the change in the commit message.

Reply:
ostree-based systems set the immutable bit on `/`.

Reply:
Hi @ssahani,

1. This is good point to change /opt on /tmp folder. Will prepare patch for it. 

2. The reason, why need to change it on /tmp/expectex, /tmp/output, under atomic host "Operation not permitted" exception occurred with any operation with this directories. 

Thanks,
Eduard